en/docs/learn/references/natural-functions.md

```diff
@@ -1,10 +1,10 @@
 # Natural Functions
 
-In this tutorial, you will learn how to use natural functions in Ballerina Integrator, which allow the function to contain instructions in natural language. 
-Such a function is evaluated at runtime with a call to an LLM. The example uses a natural function to analyze blog content to suggest a category and rate it based on predefined criteria.
+In this tutorial, you will learn how to use Natural Functions in Ballerina Integrator, which allow the function to contain instructions in natural language. 
```

Comment:
Why the change to start with an uppercase letter?

Reply:
I thought the term should be consistent in every places

Reply:
Here we are referring to it as a concept, not as a reference to something in the UI.

Reply:
Updated, Please check
https://github.com/wso2/docs-bi/pull/77/commits/14ba78fbab8d6975737959db30a8046e06c17ea8